x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/ml/change_point_agg.yml

```diff
@@ -211,3 +261,137 @@ setup:
             }
           }
   - is_true: aggregations.change_point.type.indeterminable
+---
+"Test change_point with terms":
+  - do:
+      search:
+        index: store
+        size: 0
+        body: >
+          {
+            "aggs": {
+              "terms": {
+                 "terms": {
+                   "field": "kind"
+                 },
+                 "aggs": {
+                    "date": {
+                      "date_histogram": {
+                        "field": "time",
+                        "fixed_interval": "1s"
+                      },
+                      "aggs": {
+                        "min": {
+                          "min": {
+                            "field": "cost"
+                          }
+                        }
+                      }
+                    },
+                    "change_point": {
+                      "change_point": {
+                        "buckets_path": "date>min"
+                      }
+                    }
+                 }
+              }
+            }
+          }
+  - is_true: aggregations.terms.buckets.0.change_point.type.trend_change
+  - is_true: aggregations.terms.buckets.0.change_point.type.trend_change.p_value
+  - is_true: aggregations.terms.buckets.0.change_point.type.trend_change.r_value
+  - is_false: aggregations.terms.buckets.1.change_point.type.trend_change
+  - is_false: aggregations.terms.buckets.1.change_point.type.trend_change.p_value
+  - is_false: aggregations.terms.buckets.1.change_point.type.trend_change.r_value
+
+  - do:
+      search:
+        index: store
+        size: 0
+        body: >
```

Comment:
@darnautov dig this for an example. Here it filters to include only low p_values. A p_value of 1 indicates NO change.